debug/customization/vector-tile-layer/smk-init.js

```diff
@@ -0,0 +1,298 @@
+SMK.INIT( {
+    containerSel: '#smk-map-frame',
+    config: [ './smk-config.json', '?' ]
+} )
+.then( function ( smk ) {
+    // SMK initialized
+
+    include([
+		// Load Leaflet VectorGrid
+        {
+            url: "https://unpkg.com/leaflet.vectorgrid@1.2.0",
+            loader: "script"
+        },
+		// Load EU countries JSON as 'euCountries'
+        {
+            url: "https://leaflet.github.io/Leaflet.VectorGrid/eu-countries.js",
```

Comment:
Can this eu-contries file be sourced from a CDN somewhere? If not, we should download a local copy and use it; otherwise, if this file becomes unavailable the whole demo breaks.

Reply:
I didn't find it in a CDN so I'll download it. I'll add it in a <script> in index.html; adding it here as "eu-countries.js" results in smk.js looking for it in /dist/assets/src/eu-countries.js, not finding it, and giving an error.

Reply:
I can load it like this on a Windows machine after placing a copy in the assets directyory:

```
    {
            url: "./assets/euCountries.js",
            loader: "script"
    }
```

Reply:
This works for me too. Done.